ui/src/app/components/common/datagrid/filters/datetime-range-filter/datetime-range-filter.component.ts

```diff
@@ -15,10 +15,10 @@
 
 import { AfterViewInit, Component, Input } from '@angular/core';
 import { ClrDatagridFilterInterface } from '@clr/angular';
-import { DagRunModel } from '../../../../models/dagRuns/dagRun.model';
+import { DagRunModel } from '../../../../../models/dagRuns/dagRun.model';
```

Comment:
Interesting that the filters work, even though you're not passing a `DagRunModel`. I'm thinking, if we generalize the filters, then probably the filters should not implement
`ClrDatagridFilterInterface<DagRunModel>`, but `ClrDatagridFilterInterface<any>` instead. The type parameter is only used for the `accepts` method, and there we have a type-unsafe access anyway with `item[this.property]`, i.e. we can't make this component type-safe anyway, so we don't lose anything by replacing `DagRunModel` with `any`